### ISSUE TITLE
Failed to hit the breakpoint when attach a running python.exe.

### ISSUE THREAD

**Issue by linette-zyy** [microsoft/PTVS#6932]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/159417442-92dd73fe-bef5-478b-b6aa-ec5be46ca5f3.png)

**Steps to Reproduce**
1. Launch VS and create new python application
2. Add an input('>') or raw_input('>') at the top of your code - depends on version of python
3. Add some more python code after that with breakpoint(s)
4. Start WITHOUT debugging
	a. Console will wait for you to input something
5. Debug > Attach to Process, select the appropriate python.exe (easiest if there's only one, you may want to close interactive windows first) 
6. Type enter in the console
7. Check that it stops at the breakpoint

**Expected behavior**
It stops at the breakpoint.

**Additional context and screenshots**
Failed to hit the breakpoint and an error message show in the Diagnostic Tool window
![image](https://user-images.githubusercontent.com/59192491/159417650-1b4bcb5a-ec8b-4453-ba57-2c03a3a41eb3.png)

**Diagnostics Hub output**
`Request to DataWarehouse host was unsuccessful: 'Could not load file or assembly 'Microsoft.Diagnostics.Tracing.TraceEvent, Version=2.0.73.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'`

**Comment by ttSpace** [comment]
In todsy's build, Failed to hit breakpoint while attaching running python.exe.

![image](https://user-images.githubusercontent.com/82852072/160069395-c37d4851-9e01-40bd-8dbb-351831d3a6e0.png)

![fail_to_hit](https://user-images.githubusercontent.com/82852072/160069436-2cf41ba4-1c07-4f3f-bc51-7a41a233bb20.gif)

## Logs

<details>

<summary>Output for <code>Debugr</code> in the <code>Output</code> panel 
</summary>

<p>

```
'python.exe' (Win32): Loaded 'C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\python.exe'. Symbols loaded.
'python.exe' (Win32): Loaded 'C:\Windows\System32\ntdll.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\kernel32.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\KernelBase.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbase.dll'. 
'python.exe' (Win32): Loaded 'C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\vcruntime140.dll'. 
'python.exe' (Win32): Loaded 'C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\python39.dll'. Symbols loaded.
'python.exe' (Win32): Loaded 'C:\Windows\System32\ws2_32.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\rpcrt4.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\version.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\msvcrt.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\advapi32.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\sechost.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\cryptsp.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\rsaenh.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\cryptbase.dll'. 
'python.exe' (Win32): Loaded 'C:\Windows\System32\bcryptprimitives.dll'. 
The thread 0x3a18 has exited with code 0 (0x0).
'python.exe' (Win32): Loaded 'C:\Windows\System32\kernel.appcore.dll'. 
The thread 0x1bb8 has exited with code 0 (0x0).
The thread 0x24b4 has exited with code 0 (0x0).
The thread 0x24a0 has exited with code 0 (0x0).
The program '[12884] python.exe' has exited with code 0 (0x0).
```

</p>
</details>

**Comment by int19h** [comment]
In the video, the "Code Type" field in the Attach to Process dialog was auto-filled to Native (which is itself a bug), so this is debugging native code only. If you manually change Code Type to Python, does it work?

**Comment by ttSpace** [comment]
@int19h It didn't work, still failed to hit breakpoint.

![image](https://user-images.githubusercontent.com/82852072/160315107-a51c36f1-3ba0-4530-8ad9-b8d39f968e93.png)

**Comment by int19h** [comment]
This was fixed in #6951
